piker/clearing/_client.py

```diff
@@ -263,7 +261,7 @@ async def open_ems(
                     relay_order_cmds_from_sync_code,
                     fqsn,
                     trades_stream
-                )
+                ) 
```

Comment:
looks like trailing whitespace?

You might want to tweak your editor to detect this 😉